cellular_automata/one_dimensional.py

```diff
@@ -0,0 +1,93 @@
+'''
+Returns an image of 16 generations of
+one-dimensional cellular automata 
+based on a given ruleset number
+https://mathworld.wolfram.com/ElementaryCellularAutomaton.html
+'''
+# Formats inputted ruleset
+def format_ruleset(RULE_NUM):
+    '''
+    >>> format_ruleset(11100)
+    [0,0,0,1,1,1,0,0]
+    >>> format_ruleset(0)
+    [0,0,0,0,0,0,0,0]
+    >>> format_ruleset(11111111)
+    [1,1,1,1,1,1,1,1]
+    '''
+    RULE = [int(a) for a in RULE_NUM]
+    while True:
+        if len(RULE) == 8:
+            break
+        else:
+            RULE.insert(0, 0)
+    #returns formatted ruleset
+    return(RULE)
```

Comment:
```suggestion
def format_ruleset(ruleset: int) -> list:
    '''
    >>> format_ruleset(11100)
    [0,0,0,1,1,1,0,0]
    >>> format_ruleset(0)
    [0,0,0,0,0,0,0,0]
    >>> format_ruleset(11111111)
    [1,1,1,1,1,1,1,1]
    '''
    return [int(c) for c in f"{ruleset:08}"[:8]]
```

Reply:
f"{ruleset:08}" converts ruleset into a "zero filled" string.

[:8] takes only the first eight characters.  Please add a test with 10 digits to ensure that the list that this function returns is only the first 8 digits.

Reply:
Type hints on the function signature make it clear that the function takes in an int and returns a list.